chart/values.yaml

```diff
@@ -434,3 +434,23 @@ postgresql:
   enabled: true
   postgresqlPassword: postgres
   postgresqlUsername: postgres
+# Git sync
+gitSync:
```

Comment:
We should put this under a `kubernetesExecutor` section in case we need to expand to further k8sexecutor-specific features

Reply:
I thought so too, but the  git sync sidecars might  also be get used  without the `kubernetesExecutor` ?

Reply:
Ok yeah I can see that. would just need to set a relatively fast cycle for pinging/re-pulling the repo s.t. the worker stays in sync with the scheduler/webserver.

Reply:
updated the default sync interval to a default of 1 minute, have added in  a PVC as well